### PR TITLE
Cockpit 297

### PR DESCRIPTION
**Pull request by martinpitt**
Preview: https://martinpitt.github.io/cockpit-project.github.io/blog/cockpit-297.html

### CODE REVIEW

File: _posts/2023-07-26-cockpit-297.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 297
+author: skobyda
+date: '2023-07-26'
+tags: cockpit, podman
+slug: cockpit-297
+category: release
+summary: 'Administrator can change user shell, Python bridge in fedora-39 and centos-stream-9, Show container's latest checkpoint'
```
Comment (martinpitt) [inline _posts/2023-07-26-cockpit-297.md:8]:
```suggestion
summary: 'Administrator can change user shell, Python bridge in Fedora 38 and CentOS Stream/RHEL 9, show container's latest checkpoint'
```

Comment (martinpitt) [reply _posts/2023-07-26-cockpit-297.md]: Applied, and also fixed the quoting.

File: _posts/2023-07-26-cockpit-297.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+summary: "Administrator can change user shell, Python bridge in Fedora 38 and CentOS Stream/RHEL 9, Show container's latest checkpoint"
+---
+
+Cockpit is the [modern Linux admin interface](https://cockpit-project.org/).
+We release regularly.
+
+Here are the release notes from Cockpit 297 and cockpit-podman 73:
+
+
+## Allow administrators to change the user shell
```
Comment (garrett) [inline _posts/2023-07-26-cockpit-297.md:17]:
```suggestion
## Support for changing user shells
```

File: _posts/2023-07-26-cockpit-297.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+
+Administrators can now change a users shell via the user account page.
```
Comment (garrett) [inline _posts/2023-07-26-cockpit-297.md:19]:
```suggestion
Administrators can now change user default shells on the Accounts page.
```

File: _posts/2023-07-26-cockpit-297.md [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+
+![screenshot of ## allow administrators to change the user shell]({{ site.baseurl }}/images/297-allow-administrators-to-change-the-user-shell.png)
+
+![screenshot of ## allow administrators to change the user shell]({{ site.baseurl }}/images/297-allow-administrators-to-change-the-user-shell-2.png)
+
+## Move to Python bridge in Fedora 38 and CentOS Stream 9
```
Comment (garrett) [inline _posts/2023-07-26-cockpit-297.md:25]:
```suggestion
## Use the Python bridge in Fedora 38 and CentOS Stream 9
```

File: _posts/2023-07-26-cockpit-297.md [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+
+The rewritten Python bridge that was introduced in [Cockpit 294](https://cockpit-project.org/blog/cockpit-294.html) gets enabled on Fedora 38 and CentOS Stream 9. Please [let us know](https://github.com/cockpit-project/cockpit/issues/new?template=bug_report.yml) if you run into any trouble!
```
Comment (garrett) [inline _posts/2023-07-26-cockpit-297.md:27]:
```suggestion
The rewritten Python bridge, introduced in [Cockpit 294](https://cockpit-project.org/blog/cockpit-294.html), is now used on Fedora 38 and CentOS Stream 9. Please [let us know](https://github.com/cockpit-project/cockpit/issues/new?template=bug_report.yml) if you run into any trouble!
```

File: _posts/2023-07-26-cockpit-297.md [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+
+
+## Podman: Show time of container's latest checkpoint
```
Comment (garrett) [inline _posts/2023-07-26-cockpit-297.md:30]:
```suggestion
## Podman: Show the time of the latest container checkpoint
```

File: _posts/2023-07-26-cockpit-297.md [hunk 7]
```diff
@@ -0,0 +1,49 @@
+---
+title: Cockpit 297
+author: skobyda
+date: '2023-07-26'
+tags: cockpit, podman
+slug: cockpit-297
+category: release
+summary: "Administrator can change user shell, Python bridge in Fedora 38 and CentOS Stream/RHEL 9, Show container's latest checkpoint"
```
Comment (garrett) [inline _posts/2023-07-26-cockpit-297.md:8]:
```suggestion
summary: "Support for changing user shell, Python bridge in Fedora 38 and CentOS Stream/RHEL 9, Show container's latest checkpoint"
```

**Review by garrett — CHANGES_REQUESTED** [review]
How are these changes?

File: _posts/2023-07-26-cockpit-297.md (hunk 2 above)
Comment (martinpitt) [reply _posts/2023-07-26-cockpit-297.md:17]: Please also prefix with "Users:"

**Comment by skobyda** [comment]
All issues fixed

**Review by skobyda — APPROVED** [review]

**Pull request merged**